lib/temporalio/worker/sync_worker.rb

```diff
@@ -21,66 +21,66 @@ def initialize(core_worker)
       def poll_activity_task
         with_queue do |done|
           core_worker.poll_activity_task do |task, error|
-            done.call(task && Coresdk::ActivityTask::ActivityTask.decode(task), error)
+            done.call(task && Temporalio::Bridge::Api::ActivityTask::ActivityTask.decode(task), error)
           end
         end
       end
 
       def complete_activity_task_with_success(task_token, payload)
-        result = Coresdk::ActivityResult::ActivityExecutionResult.new(
-          completed: Coresdk::ActivityResult::Success.new(result: payload),
+        result = Temporalio::Bridge::Api::ActivityResult::ActivityExecutionResult.new(
+          completed: Temporalio::Bridge::Api::ActivityResult::Success.new(result: payload),
         )
 
         complete_activity_task(task_token, result)
       end
 
       def complete_activity_task_with_failure(task_token, failure)
-        result = Coresdk::ActivityResult::ActivityExecutionResult.new(
-          failed: Coresdk::ActivityResult::Failure.new(failure: failure),
+        result = Temporalio::Bridge::Api::ActivityResult::ActivityExecutionResult.new(
+          failed: Temporalio::Bridge::Api::ActivityResult::Failure.new(failure: failure),
         )
 
         complete_activity_task(task_token, result)
       end
 
       def complete_activity_task_with_cancellation(task_token, failure)
-        result = Coresdk::ActivityResult::ActivityExecutionResult.new(
-          cancelled: Coresdk::ActivityResult::Cancellation.new(failure: failure),
+        result = Temporalio::Bridge::Api::ActivityResult::ActivityExecutionResult.new(
+          cancelled: Temporalio::Bridge::Api::ActivityResult::Cancellation.new(failure: failure),
         )
 
         complete_activity_task(task_token, result)
       end
 
       def record_activity_heartbeat(task_token, payloads)
-        proto = Coresdk::ActivityHeartbeat.new(
+        proto = Temporalio::Bridge::Api::CoreInterface::ActivityHeartbeat.new(
           task_token: task_token,
           details: payloads,
         )
-        encoded_proto = Coresdk::ActivityHeartbeat.encode(proto)
+        encoded_proto = Temporalio::Bridge::Api::CoreInterface::ActivityHeartbeat.encode(proto)
```

Comment:
Verify why the extra `CoreInterface` here

Reply:
I think if we use `Temporalio::Bridge::Core::` as the prefix for the core API it would be a bit nicer.